### DISCUSSION TITLE
build expressions using multiple `searches`

### DISCUSSION THREAD

**Discussion by hrodmn** [stac-utils/titiler-pgstac#189] [Ideas]
It would be really cool to be able to generate visualizations that use assets from multiple `pgstac` searches. For example:
- showing differences in NDVI or land cover classification from a single collection at different points in time
- generating expressions that combine assets from multiple collections (e.g. DEM + land cover)

It would be very powerful because the different data sources might have different extents/resolutions but the arrays created by the mosaic readers for methods like `tile` would be compatible.

**Comment by vincentsarago** [comment]
sounds like layer composition! 

Sadly, I don't see a straight forward path to do this 😓
